wv2util/MainWindow.xaml.cs

```diff
@@ -234,6 +241,18 @@ private void Hyperlink_RequestNavigate(object sender, RequestNavigateEventArgs e
             e.Handled = true;
         }
 
+        private void Hyperlink_ExecutedRouted(object sender, ExecutedRoutedEventArgs e)
+        {
+            try

```

Comment:
Can you please have this method call the other Hyperlink_RequestNavigate method that is doing kind of the same thing? Or have them both call into the same helper?